spec/mailers/user_mailer_spec.rb

```diff
@@ -432,4 +432,74 @@
       end
     end
   end
+
+  describe "invited to collection notification" do
+    let(:collection) { create(:collection) }
+    let(:otheruser) { create(:user) }
+    let(:work) { create(:work, authors: [otheruser]) }
```

Comment:
I think this needs to be a pseud, e.g. `user.pseuds.first`.

Reply:
Yes, I left the tests to run while I was sleeping, fixed now :)